BudgetSquirrel.Api/ClientApp/src/responsive.scss

```diff
@@ -0,0 +1,11 @@
+@media (min-width: 1920px) {
+  main {
+    padding-left: 320px;
```

Comment:
Ok, I'll change this in my next PR. I would like to get a lot of this stuff merged since there's some infrastructure stuff we'll need for pretty much everything moving forward.